frontend/src/components/Contributions/ContributionForm.vue

```diff
@@ -50,15 +50,28 @@
           <span v-for="error in errors" class="errors" :key="error">{{ error }}</span>
         </b-col>
       </validation-provider>
-      <label class="mt-3">{{ $t('form.amount') }} {{ $t('math.asterisk') }}</label>
-      <b-input-group size="lg" prepend="GDD">
-        <b-form-input
-          id="contribution-amount"
-          v-model="form.amount"
-          type="text"
-          :formatter="numberFormat"
-        ></b-form-input>
-      </b-input-group>
+      <validation-provider
+        :name="$t('form.amount')"
+        :rules="{
+          required: true,
+          gddSendAmount: [0.01, balance],
```

Comment:
Balance is not defined. We need the max available amount for the selected month here. This should be available in the parent component and must be past as a property. Or you pass both max amounts for the two possible months to this component ...

Reply:
ok. done.
the maximum betag is calculated in the computed validMaxGDD. 
I would first leave the data in the ContributionForm and let it be calculated. 
since we are changing a lot graphically ... this will become a central method in the future.